src/main/java/mate/academy/webapp/dto/book/CreateBookRequestDto.java

```diff
@@ -16,8 +16,8 @@ public class CreateBookRequestDto {
     private String author;
     @NotNull(message = NOT_NULL)
     private String isbn;
-    @NotNull(message = GREATER_THAN_ZERO)
-    @Positive
+    @NotNull(message = NOT_NULL)
+    @Positive(message = GREATER_THAN_ZERO)
```

Comment:
That's a bit confusing, you already have these changes in the main branched, haven't you?